submissions/morenko/oop-exercise/index.js

```diff
@@ -0,0 +1,86 @@
+// создадим родительский класс World
+class WorldInhabitant {
+    // обьявляем поля (свойства)
+    
+    // объявляем метод конструктора (ожидаем на входе значения)
+    constructor(name, species, gender, saying){
+        // инициализация полей
+        this.name = name;
+        this.species = species;
+        this.gender = gender;
+        this.saying = saying;
+    }
+  // объявление метода toString()
+  // toString() возвращает строку, представляющую объект
+  // обращение к собственному атрибуту через this
+    toString() {
+       return `${this.name}; ${this.species}; ${this.gender}; ${this.saying}; `;
+    }
+  }
+  
+  // создадим два класса наследующие от родительского часть свойств
+  class Human extends WorldInhabitant {
+    constructor(name, species, gender, legs, hands, saying) {
```

Comment:
This is a little strange, the class is already name `Human`, why would you pass `species` name in the `Human` class? Are there different species of humans? Try to fix this.

Reply:
Also, humans always have 2 arms and 2 legs, aren't they? Why would you pass it here?